src/error.h

```diff
@@ -43,6 +43,13 @@ typedef struct {
 #define ASSERT(cond, ...) \
   ((cond) ? (void)(0) : (raise_fatal_error_impl(__FILE__, PRETTY_FUNCTION, __LINE__, false, false, #cond, "" __VA_ARGS__), BUILTIN_UNREACHABLE()))
 
+#ifdef NDEBUG
+# define DBGASSERT(...)
+#else
+# define DBGASSERT(cond, ...) \
+  ((cond) ? (void)(0) : (raise_fatal_error_impl(__FILE__, PRETTY_FUNCTION, __LINE__, false, false, #cond, "" __VA_ARGS__), BUILTIN_UNREACHABLE()))
```

Comment:
Is there any reason why this isn’t just defined like so in the `#else` part?
```c
#    define DBGASSERT(...) ASSERT(__VA_ARGS__)
```

Reply:
Nope; that's probably a good idea haha